inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/cls/ClsSinkContext.java

```diff
@@ -104,26 +97,25 @@ public void reload() {
                 }
             });
 
-            SortTaskConfig newSortTaskConfig = SortClusterConfigHolder.getTaskConfig(taskName);
+            SortTaskConfig newSortTaskConfig = SortConfigHolder.getTaskConfig(taskName);
             if (newSortTaskConfig == null || newSortTaskConfig.equals(sortTaskConfig)) {
                 return;
             }
             LOG.info("get new SortTaskConfig:taskName:{}:config:{}", taskName,
                     new ObjectMapper().writeValueAsString(newSortTaskConfig));
             this.sortTaskConfig = newSortTaskConfig;
```

Comment:
If requestNodeConfig.getVersion() < clsNodeConfig.getVersion(), reload function can directly return.

Reply:
Both NodeConfig and DataflowConfig have their version. Must check all versions of dataflow and node.

Reply:
The config update problem will be fixed in #10224